hooman/demos/escape_the_obstacles.py

```diff
@@ -0,0 +1,90 @@
+import random
+import pygame
+from hooman import Hooman
+
+# Initialize pygame and hooman
+pygame.init()

```

Comment:
We don't need this line in Hooman

Reply:
@Abdur-rahmaanJ , you are specifying pygame.init() right? I will remove it and commit again.

Reply:
Yes!